libcloud/loadbalancer/drivers/elb.py

```diff
@@ -115,6 +124,14 @@ def get_balancer(self, balancer_id):
         data = self.connection.request(ROOT, params=params).object
         return self._to_balancers(data)[0]
 
+    def get_tags(self, balancer_id):
```

Comment:
would you ever want to call `get_tags` without calling `list_balancers`? If not, it would be better to leave this as an internal method and remove it from the base class.